src/main/java/com/redhat/cloud/policies/engine/process/PoliciesAction.java

```diff
@@ -21,12 +21,27 @@
 
 public class PoliciesAction {
     private String accountId;
+    private String orgId;
     private LocalDateTime timestamp;
     private Context context = new Context();
     private Set<Event> events = new HashSet<>();
 
+    private final boolean useOrgId;
+
+    public PoliciesAction() {
+        this(false);
+    }
+
+    public PoliciesAction(boolean useOrgId) {
+        this.useOrgId = useOrgId;
+    }
+
     public String getKey() {
-        return accountId + context.systemCheckIn;
+        if (useOrgId) {
+            return orgId + context.systemCheckIn;
+        } else {
+            return accountId + context.systemCheckIn;
+        }
     }
```

Comment:
This method isn't used anymore, it's a relic from the past. It can be deleted.

As a consequence, I don't think we need to pass the `useOrgId` boolean to the constructor.

Reply:
But the getKey() method would not work anymore when there is no flag present

Reply:
this one: https://github.com/RedHatInsights/policies-engine/pull/377/files#diff-54a89ce45518be890cf1a0c32d4002540420853123cbf94b647e5fb638bcd4dcR39

Reply:
Yes but it's not used anymore. The method can be safely deleted.